src/parser/parser_utils.cpp

```diff
@@ -25,55 +25,33 @@ namespace parser {
 
 std::string indent(uint num_indent) { return std::string(num_indent, '\t'); }
```

Comment:
Don't use tabs. Use spaces.

Reply:
Actually this was not written by me, so do you prefer 2 spaces instead?

Reply:
Use `GETINFO_SPACER`:

https://github.com/cmu-db/peloton/blob/master/src/include/util/string_util.h#L25

Reply:
Please take a look at my latest commit, if you like that we can just use that.

Reply:
Good enough.